kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -619,8 +616,9 @@
         }
       },
       searchTerms(newVal, oldVal) {
-        if (!isEqual(newVal, oldVal) && this.displayingSearchResults) {
-          this.$router.push({ ...this.searchLink, sidePanel: false });
+        if (!isEqual(newVal, oldVal)) {
+          this.$router.push({ ...this.searchTabLink });
```

Comment:
Getting redundant navigation errors here now - simplest fix might be to just to update it to:
```suggestion
          if (!isEqual(this.searchTabLink, this.$route)) {
            this.$router.push({ ...this.searchTabLink });
          }
```

Don't know if this will actually work, alternatively can just wrap it in a try/catch.